jwala-common/src/main/java/com/cerner/jwala/common/rule/jvm/JvmNameRule.java

```diff
@@ -24,7 +24,7 @@ public boolean isValid() {
 
     @Override
     protected String getMessage() {
-        return name != null && name.length() > MAX_LENGTH ? "Length of JVM should not exceed 64 characters" : "Invalid Jvm Name : \"" + name + "\"";
+        return name != null && name.length() > MAX_LENGTH ? "Length of JVM should not exceed "+ MAX_LENGTH+ " characters" : "Invalid Jvm Name : \"" + name + "\"";
```

Comment:
Please wrap the line if it's too long. 125 characters is the de facto standard right now so what I do is set my editor's guideline at 120 character so I would know when to wrap. If it exceeds a little from 120 it is fine.

![image](https://cloud.githubusercontent.com/assets/10065908/25681914/18ba4910-3024-11e7-8951-ad8c85616185.png)